Toolbox/final/hashes.txt

```diff
@@ -1,8 +1,12 @@
 FileName = "finalScript.sh"
-FileSize = "1266"
-CheckSum = "e26f12b8d3319fcfeb6417f47f2aadd70914fd1e"
+FileSize = "1063"
+CheckSum = ""5478e2177180c1654b2fbe2ea307078eb1ea0a26"
```

Comment:
Checksum has two " in the beginning. Also in line 7 and 11. Looks wrong to me.

Reply:
Wow, nice catch - turns out I need another fix on my update tool. Strangely it still works...